### ISSUE TITLE
Horizontal block inputs are centered instead of aligned with icons

### ISSUE THREAD

**Issue by RoboErikG** [LLK/scratch-blocks#1101]
### Expected Behavior

From [the spec](https://github.com/LLK/scratch-blocks/wiki/Design) it looks like the icon and dropdown should be 4 units (16 pixels) from the left edge and 2 units (8 pixels) from the right edge of the block. The spec doesn't give the dropdown position explicitly, so Carl will need to confirm.

### Actual Behavior

Currently, the dropdown/input is centered with 3 units on either side, making it offset from the icons on the blocks.
![image](https://user-images.githubusercontent.com/11066279/30939900-fdf2ca42-a393-11e7-9a00-a7d9f3445f4b.png)

### Steps to Reproduce

Check rendering of horizontal blocks in [the playground.](http://llk.github.io/scratch-blocks/playgrounds/tests/horizontal_playground.html?dir=ltr&toolbox=categories&side=bottom)

### Operating System and Browser

Ubuntu with Chrome

**Comment by carljbowman** [comment]
@RoboErikG - Always good to check on this stuff. It looks like the input is rendering correctly in that screenshot.

We chose to optically align some of the elements within the horizontal grammar. Icon is offset slightly (optically aligned) to allow the notches to stand out and the icon to have some space allowing the eye to parse it.

The input is aligned with the block rather than the icon. We did some explorations of moving the input over another 4px (1unit) to the right, but it started to cause visually dense focal point on the bottom-right corner of the block.

Here is a spec:

![r4_standard-int-input-block](https://user-images.githubusercontent.com/3409578/31127489-141f7d78-a81d-11e7-85c2-25664bdc37ad.png)
